src/components/ImportForm/utils/submit-utils.ts

```diff
@@ -99,6 +101,14 @@ export const createResources = async (formValues: ImportFormValues, strategy: Im
 
   await createSecrets(importSecrets, namespace, false);
 
+  const integrationTestValues: IntegrationTestFormValues = {
+    name: `${applicationName}-enterprise-contract`,
+    pipeline: 'enterprise-contract',
+    bundle: 'quay.io/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract:devel',
```

Comment:
Idea: Maybe the default bundle url and the pipeline name could be placed in separate constants somewhere.

Reply:
+1 I'd love that. I'm just not familiar enough with this code base to tell where is a good place for it.

Reply:
@lcarva  You could create a const file in integrationTest utils folder ([IntegrationTest/IntegrationTestForm/utils](https://github.com/openshift/hac-dev/tree/main/src/components/IntegrationTest/IntegrationTestForm/utils)) and place these constants there  and you can also move this `integrationTestValues` const to that new location and import this here in submit-utils.

Reply:
Done. Thanks!